point.json

```diff
@@ -0,0 +1 @@
+{"accuracy":0,"alt":2279.5171,"course":280,"group_id":"3690","id":"05da255d-9079-4a9f-af78-ac7463aab412","is_active":false,"lat":39.705635,"lon":-105.27304367,"speed":30.3708,"thing":{"external_id":"410037000c47373336373936","id":"ead04cde-fc91-42c1-af6e-2a6d7440292a"},"timestamp":"2022-02-23T21:19:24.00Z","type":"point"}
```

Comment:
"thing"??

Reply:
yeah they use thing to denote whatever the object that is sending this message is. It could be an engine, station, etc.

Reply:
can this be enumerated or slightly less generic? like data?